css_advanced/electives-style.css

```diff
@@ -0,0 +1,99 @@
+.all-contents {
+  display: flex;
+  flex-direction: column;
+}
+
+h2 {
+  color: #97bf0e;
+  margin: 0px;
+}
+
+.course-info {
+  font-style: italic;
+}
+
+.button-shape {
+  border: 1px solid #c3c3c2;
+  border-radius: 10px;
+  height: 30px;
+}
+
+button[name="login"],

```

Comment:
You may want to change the mouse pointer to hand on hover of the mouse over all the buttons.